mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -1,4 +1,147 @@
 CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
+	COMMENT A registry is used to register various in-game components. Almost all parts of the
+	COMMENT game - from blocks, items, and entity types, to cat types, goat horn instruments,
+	COMMENT and structure pools - are registered in registries. Registry system allows the game
+	COMMENT to enumerate all known types of something, and to assign a unique identifier to each
+	COMMENT of those. Therefore, registering an object in the registry plays a very important
+	COMMENT role, and failure to register new instances of registerable object usually results
+	COMMENT in a bug or even a crash.
+	COMMENT
+	COMMENT <h2 id="terms">Terminologies</h2>
+	COMMENT <p>A <strong>registry</strong> is an object that holds the mapping between three things:
+	COMMENT the string ID, the numeric ID, and the registered value. There are many registries
+	COMMENT for different types of registerable objects, and a registry's type parameter indicates
+	COMMENT the accepted type. For example, you register your {@link Block} to {@code
+	COMMENT Registry<Block>}. It's important to note that registries themselves are registered
+	COMMENT in a "registry of registries", {@link #ROOT}.
+	COMMENT
+	COMMENT <p>The <strong>string ID</strong>, usually just called "ID", is a human-readable
+	COMMENT {@link Identifier} that uniquely identifies the registered value in a registry.
+	COMMENT This should stay the same between two game versions, and is usually used for disk
+	COMMENT storage.
+	COMMENT
+	COMMENT <p>The <strong>numeric ID</strong> or <strong>raw ID</strong> is an integer
+	COMMENT assigned automatically by the registry to each registered value. This is not
+	COMMENT guaranteed to stay the same between two game versions, and is usually used for
+	COMMENT networking purposes.
+	COMMENT
+	COMMENT <p>The <strong>registered value</strong>, often just called "value" in the code,
+	COMMENT is the value added to the registry. The registry's type parameter determines
+	COMMENT the type of the registered value.
+	COMMENT
+	COMMENT <p>Each registered value can also be identified with a <strong>{@linkplain RegistryKey
+	COMMENT registry key}</strong>. A registry key is a combination of the registry's ID and
+	COMMENT the registered value's ID. Using a registry key makes the type of the ID's
+	COMMENT associated value clear, as the type parameter contains the type.
+	COMMENT
+	COMMENT <p>A <strong>{@linkplain RegistryEntry registry entry}</strong> is an object
```

Comment:
Is this critical to registry? I recall it was only added during the tag refactors, that it can optionally refer to registry elements but is not essential of registry usage. If nonessential, I recommend moving the description to the class itself and briefly mention this as a utility.

Reply:
At least we need some explanation on intrusive registry entry, that's a main crasher we see in Fabricord. Maybe we can move detailed javadoc to the class (the same javadoc also exists there), but for now this is fine.